packages/backend/discovery/starknet/discovered.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "starknet",
```

Comment:
Revert changes to this file unless `callProxy` was somehow breaking things

Reply:
yup, callImplementation was breaking things